src/main/java/enums/Currency.java

```diff
@@ -0,0 +1,16 @@
+package enums;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+
+@Getter
+@RequiredArgsConstructor
+@ToString
```

Comment:
Could you put the decorators from shortest to the longest? Not required to merge the PR tho:)

Reply:
Now it looks a little cleaner 💧 , thanks